languages/csharp/exercises/concept/switch-statements/.docs/hints.md

```diff
@@ -0,0 +1,25 @@
+## General
+
+[switch statement][switch-statement] documentation provides an introduction to `switch` statements.
+
+### 1. Output descriptions of the players based on their shirt number
+
+The [`break`][break] statement is useful.
```

Comment:
```suggestion
- The [`break`][break] statement is useful.
```

Reply:
I feel that the rule we established was that we use bullet points if there is more than one hint per task.  But, as long as I just have to click a button I am happy enough.